modules/Settings/PickListDependency/views/List.php

```diff
@@ -13,7 +13,7 @@ class Settings_PickListDependency_List_View extends Settings_Vtiger_List_View
 	public function preProcess(\App\Request $request, $display = true)
 	{
 		$moduleModelList = Settings_PickListDependency_Module_Model::getPicklistSupportedModules();
-		$forModule = $request->getByType('formodule', 'Alnum');
+		$forModule = $request->get('formodule');
```

Comment:
We recomend use function isEmpty

$forModule = '';
if (!$request->isEmpty('formodule')) {
			$forModule = $request->getByType('formodule', 'Alnum');
}

Reply:
done